src/tests/unit/tests/DetailsView/reports/components/report-sections/__snapshots__/report-body.test.tsx.snap

```diff
@@ -180,6 +180,7 @@ exports[`ReportBody renders 1`] = `
         "violations": Array [],
       }
     }
+    toUtcString={[Function]}
```

Comment:
Did this snapshot change leak from a different code change somewhere else?

Reply:
Yes, I think it was a merge conflict.